Sample/Kitchen Sink/DemonstrationTools/UnitConverter.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import Foundation
+import SwiftUI
+import FlightUI
+
+struct UnitConverter: View {
+    @EnvironmentObject var theme: Theme
+    @StateObject var themeManager = ThemeManager(current: .dark)
+    @ObservedObject var demonstrationVM = UnitConverterViewModel()
+    @State private var weightValuesSwapped = false
+
+    var body: some View {
+        ScrollView {
+
+            VStack {
```

Comment:
Future Improvement/Opinion: Grouping could be done a bit cleaner based on the resultant UI, so instead of wrapping each input field in it's own view (since it already is a view), I would have grouped together these stacks.

This would look something like:

```
var body: some View {
    ScrollView {
        VStack {
            HeadingView(weight...)
            weightConverter
            HeadingView(adjustable...)
            lengthInputConverter
            lengthOutputConverter
        }
    }
}

var weightConverter: some View {
    HStack {
        InputField(...).textFieldStyle(...)
        InputField(...).textFieldStyle(...)
        Button(...).buttonStyle(...)
    }
}
```